### PR TITLE
Raven cleanup mafToFa

### PR DESCRIPTION
**Pull request by RavenYL**
- Added testing coverage for mafToFa
- I found that I can only use nano to edit the mafs, not my text editor Atom, which will delete an empty line at the end of the file and cause a fatal error

### CODE REVIEW

**Review by craiglowe — DISMISSED** [review]
Wonderful!

File: cmd/mafToFa/testdata/test.mfa [hunk 1]
```diff
@@ -0,0 +1,10 @@
+>hg38
```
Comment (rimangan) [inline cmd/mafToFa/testdata/test.mfa:1]: multiFas take the filename extension .fa

File: cmd/mafToFa/mafToFa_test.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	noMask           bool
+}{
+	{"testdata/test1.maf", "testdata/test.ref.fa", "testdata/test.species.list", "testdata/test.mfa", false},
+	{"testdata/test2.maf", "testdata/test.ref.fa", "testdata/test.species.list", "testdata/test.mfa", true},
+}
+
+func TestMafToFa(t *testing.T) {
+	var err error
+	for _, v := range mafToFaTests {
+		mafToFa(v.mafFile, v.refFile, v.speciesFile, "outFile_tmp.mfa", v.noMask)
```
Comment (cshoben) [inline cmd/mafToFa/mafToFa_test.go:24]: pinging off Riley's comment about the .mfa   I think this tmp outfile will also need it's extension changed to .fa

**Review by dasnellings — APPROVED** [review]

**Review by craiglowe — APPROVED** [review]
Great work!

**Review by rimangan — APPROVED** [review]

**Review by cshoben — APPROVED** [review]
Awesome!

**Pull request merged**